identity-engine/embedded-sign-in-widget/server/server.go

```diff
@@ -61,15 +61,16 @@ type PKCE struct {
 }
 
 type Server struct {
-	config       *config.Config
-	tpl          *template.Template
-	sessionStore *sessions.CookieStore
-	ViewData     ViewData
-	cache        *cache.Cache
-	svc          *http.Server
-	address      string
-	pkce         *PKCE
-	state        string
+	config            *config.Config
+	tpl               *template.Template
+	sessionStore      *sessions.CookieStore
+	ViewData          ViewData
+	cache             *cache.Cache
+	svc               *http.Server
+	address           string
+	pkce              *PKCE
+	state             string
+	interactionHandle string
```

Comment:
We need to save the interaction_handle so that we can re-render the widget with the same handle, if backend returns `interaction_required` error